### ISSUE TITLE
TinyMCE 4 : Cursor and Data Entry Issue

### ISSUE THREAD

**Issue by vindubey05** [tinymce/tinymce#2498]
I have to enter data, the html of which, is something like this -
< p > ( This is something data <span class='number'>1000</span>)< /p >

On the Editor, the data looks-
(This is something data 1000)

If I place my cursor here just before ) and enter data ( for example - ,1001), the data is going into the span tag. Example -
< p > ( This is something data < span class='number'>1000,1001< /span >)< /p >
I don't want this behavior. I want data to entered in the < p > tag and not < span > tag i.e. This is what i want -
< p > ( This is something data < span class='number'>1000< /span >,1001)< /p >

When I place the cursor before ")", the data should go to < p > and not < span >

Is this a feature or a bug? If there is any workaround what is it?

This is causing issues in our application.

**Comment by spocke** [comment]
Browsers leans towards the closest element we also normalize the behavior to be this way since it makes more sense. If they place the caret at <b>abc</b>| they get normalized to <b>abc|</b>.

**Comment by vindubey05** [comment]
Thank You for your input. We are using tinymce 3 and we use Mozilla extensively. In Mozilla, this isn't the behavior < p > ( This is something data < span class='number'>1000,1001< /span >)< /p >

In this case, if I place my cursor here just before ) and enter data ( for example - ,1001), the data goes to the <p> tag and not <span> tag.

We have structured our application according to this and now when we upgraded it to tinymce 4, the data is now going to <span> tag and not  <p> tag.

Could this be a bug in the tinymce version 3 for Mozilla?

Is there any way ( any customization or settings) to change this behavior so that data goes to <span> tag?
